OneCommunityGlobal/HighestGoodNetworkApp
# eslint fixing PermissionsManagement component

# Description
Fix linting for the PermissionsManagement component

## Related PRS (if any):
#1190 Previous developer left this PR so I took over it.

## Main changes explained:
- removed the PermissionsManagement component directory from .eslintignore and .prettierignore

## How to test:
- check into current branch
- do npm install and npm run lint to make sure you get no lint errors
- run npm run start:local and start the backend as usual
- go to Other Links -> Permissions Management and make sure everything is working (please do not add/remove any permissions).

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/5071040/dd0cb4d0-fdaf-4535-a5f3-936452421ad3)
